connaisseur/validators/__init__.py

```diff
@@ -0,0 +1 @@
+pass
```

Comment:
Is this necessary?

Reply:
the `pass` or the file itself? `pytest` complains that it can't find the `connaisseur.validators` module, if there is no `__init__.py` defined. so i added them for all subdirectories :shrug:

Reply:
The `pass`; it's fine for `__init__.py` to be empty. Is there any issue with it?